src/atoms/Icon.tsx

```diff
@@ -0,0 +1,107 @@
+import React, { Component } from 'react'
+import {
+    findIconDefinition,
+    IconDefinition,
+    IconLookup,
+    IconName,
+    IconPrefix,
+    SizeProp,
+    library
+} from '@fortawesome/fontawesome-svg-core'
+import { fab } from '@fortawesome/free-brands-svg-icons'
+import { far } from '@fortawesome/free-regular-svg-icons'
+import { fas } from '@fortawesome/free-solid-svg-icons'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { IconSizes, Styles } from '../constants/enum'
+
+library.add(fab, far, fas)
+
+interface IconProps {
+    className?: string
+    name: IconName
+    size?: IconSizes
+    style?: Styles
+}
+
+class Icon extends Component<IconProps> {
+    public static defaultProps = {
+        className: '',
+        size: '',
+        style: Styles.Solid
+    }
+

```

Comment:
@desweb Add two static props here which will be the IconSizes enum and the Style enum. It allows the user to use the atom without requiring the enum (and we will just export the components for the user). Look at what I did for the button and how I used these props in the DemoButton component ;)

After this change, everything will be ok for me